docs/cookbook/platform-sh.rst

```diff
@@ -151,11 +153,11 @@ This file will load ``mysql`` and ``redis`` on your Platform.sh server.
 
     # .platform/services.yaml
     mysql:
-        type: mysql
-        disk: 2048
+        type: mysql:10.0
```

Comment:
Doesn't this value go along with `sever_version` of doctrine dbal? And we have [5.5](https://github.com/Sylius/Sylius/blob/master/app/config/config.yml#L35) right now in a config file.

Reply:
Hm, good catch. I will check that, as I have `server_version` at 5.6. I'll check that one tomorrow.